smc-command/smc.c

```diff
@@ -644,6 +809,7 @@ void usage(char* prog)
     printf("%s [options]\n", prog);
     printf("    -f         : fan info decoded\n");
     printf("    -t         : list all temperatures\n");
+    printf("    -o <format>: output format for -f and -t. Options are 'json' and 'text'. Default is 'text'");
```

Comment:
```suggestion
    printf("    -o <format>: output format for -f and -t. Options are 'json' and 'text'. Default is 'text'\n");
```
Suggested fix for inconsistent output formatting.